src/target/llvm/codegen_cpu.cc

```diff
@@ -606,7 +606,9 @@ void CodeGenCPU::UnpackClosureData(TypedPointer cdata, const Array<Var>& vfields
     llvm::Type* field_type = cdata.type->getStructElementType(i);
     llvm::Value* field_addr =
         builder_->CreateInBoundsGEP(cdata.type, cdata.addr, {ConstInt32(0), ConstInt32(i)});
-    (*vmap)[vfields[i].get()] = builder_->CreateLoad(field_type, field_addr);
+    llvm::Value* load = builder_->CreateLoad(field_type, field_addr);
```

Comment:
You can add the name as the 3rd argument to `CreateLoad`.

Reply:
Thank you, that definitely makes it cleaner.  Updated here and in CreateParallelLaunch.